.travis.yml

```diff
@@ -51,6 +51,17 @@ jobs:
                 openjdk:8-jdk-alpine \
                 ./gradlew --no-daemon testcontainers:test --tests '*GenericContainerRuleTest' --scan
 
+    - env: [ NAME=docker-in-mongodb ]
+      script:
+          - |
+              DOCKER_HOST=unix:///var/run/docker.sock DOCKER_TLS_VERIFY= docker run -t --rm \
+              -v "$HOME/.gradle":/root/.gradle/ \
+              -v /var/run/docker.sock:/var/run/docker.sock \
+              -v "$(pwd)":"$(pwd)" \
+              -w "$(pwd)" \
+              openjdk:8-jdk-alpine \
+              ./gradlew mongodb:integrationTest --scan --no-daemon --no-build-cache
+
```

Comment:
Even though an integrationTest task is ran within a test stage with env: [ NAME=modules ], I added an additional test stage with env: [ NAME=docker-in-mongodb ] to test a difference in local Docker and local Docker host running tests from inside a container with mapping the Docker socket.

The table below shows such difference in detail:

| Difference  | local Docker example | local Docker host running tests from inside a container with mapping the Docker socket example |
| --- | --- | --- | 
| a host string to initialize a replica set  |  localhost:27017 |  172.17.0.1:33542 |
| a url to use with a Java Mongo driver  | mongodb://localhost:33538/test  |  mongodb://172.17.0.1:33542/test |

Reply:
ok, I'll remove this test

Reply:
done

Reply:
LGTM